lib/ExportsStorage.js

```diff
@@ -52,6 +52,17 @@ export default class ExportsStorage {
 
             this.db.run(
               `
+                CREATE VIRTUAL TABLE exports_fts USING fts4(
```

Comment:
Can you elaborate a little why the virtual table is necessary here? I could see this being implemented using "simple" `WHERE name LIKE 'foobar%'` as well. The redundancy is probably okay, I'm asking more because I'm (passively) searching for alternatives to sqlite3. Multiple people have reported that it's hard to install (e.g. https://github.com/Galooshi/atom-import-js/issues/15, https://github.com/Galooshi/atom-import-js/issues/18) and it would be easier if there was a "pure" javascript alternative.